desktop-app/src/renderer/components/FileUploader/FileUploader.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { FileUploader, FileUploaderProps } from './FileUploader';
+import { useFileUpload } from './hooks';
+
+jest.mock('./hooks');
+
+const mockHandleFileUpload = jest.fn();
+const mockHandleUpload = jest.fn();
+const mockResetUploadedFile = jest.fn();
+
+describe('FileUploader', () => {
```

Comment:
Really appreciate the unit tests! 🫡

Reply:
I wanted to add more on the ManageSuitesTool but I am having issues with the electron store, I can't seem to mock it. Do you have any idea how to mock it properly?

This is the error:
```
FAIL  src/renderer/components/DeviceManager/PreviewSuites/ManageSuitesTool/ManageSuitesTool.test.tsx
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'store')

      4 | import { sanitizeSuites } from './utils';
      5 |
    > 6 | const activeDeviceIds: string[] = window.electron.store.get(
        |                                                   ^
      7 |   'deviceManager.activeDevices'
      8 | );
      9 |

```

Reply:
Maybe we need to add a mock `electron` object to the window before the test runs? 

Something like this:

electron-mock.ts
```
const store = {};
window.electron = {
  store: {
    get: (key: string) => {
        return store[key];
    },
    set: (key: string, val: any) => {
       store[key] = val;
    }
  }
};
```

And then import this file at the top of the test file?

(haven't thought this through, just a quick suggestion)